scripts/install.sh

```diff
@@ -1,23 +1,72 @@
 #!/bin/bash
 set -e
 
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
+
 # Dafault inputs.
-KEEP_ETHEREUM_PASSWORD_DEFAULT="password"
+KEEP_ACCOUNT_PASSWORD_DEFAULT="password"
+CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY=""
```

Comment:
Yes, you're right about `CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY`. After taking a closer look I decided to split `install.sh` to 2 separate scripts which each will be responsible for its own chain, ie. [install.sh](https://github.com/keep-network/keep-core/pull/2309/commits/452ca2a4de0240a78c96e057a32b8026de24e42d) as originally will handle `Ethereum` and I will create a new `install-celo.sh` for `Celo`. 
Developing in bash is not the same experience as in `Go` and I feel like having multiple 'if' statements and handling env vars clutters this script and it will be harder to read and maintain it in the future.